stats.php

```diff
@@ -56,27 +56,22 @@
 // Get forum permissions
 $unviewableforums = get_unviewable_forums(true);
 $inactiveforums = get_inactive_forums();
-$fidnot = '1=1 AND ';
-$unviewableforumsarray = array();
+$unviewablefids = $inactivefids = array();
+$fidnot = '';
+
 if($unviewableforums)
 {
-	$fidnot .= "fid NOT IN ($unviewableforums) AND ";
-	$unviewableforumsfids = explode(',', str_replace("'", '', $unviewableforums));
-	foreach($unviewableforumsfids as $fid)
-	{
-		$unviewableforumsarray[] = (int)$fid;
-	}
+	$fidnot .= "AND fid NOT IN $unviewableforums";
```

Comment:
Where? If you mean around the `$unviewableforums`, it's just a regular PHP syntax we're using in a lot of places in MyBB.